src/SFA.DAS.Notifications.Messages/SFA.DAS.Notifications.Messages.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.1</TargetFramework>
```

Comment:
Why netstandard 1.1?

Reply:
.NET Framework 4.5 only supports using .net standard libraries targeting .NET Standard 1.0 or 1.1

Reply:
We shouldn't have any net framework 4.5 projects either 😁